starport/templates/typed/basic/stargate.go

```diff
@@ -0,0 +1,23 @@
+package basic
```

Comment:
I'm not sure if `basic` is self-explanatory with the fact that no modification to the app state is performed. We could also think about a model name like `stateless`, `declaration`, `dry`

Reply:
solved by 9204e56.